lib/features/push_notification/presentation/controller/web_socket_controller.dart

```diff
@@ -32,13 +34,15 @@ class WebSocketController extends PushBaseController {
   WebSocketChannel? _webSocketChannel;
   Timer? _webSocketPingTimer;
   StreamSubscription? _webSocketSubscription;
+  Debouncer<StateChange?>? _stateChangeDebouncer;
```

Comment:
```suggestion
  Debouncer<StateChange>? _stateChangeDebouncer;
```

Reply:
The `Debouncer` constructor needs an initial value. So we need a nullable value of `StateChange`. But the handler checks if StateChange == null, we will ignore it.